README.md

```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
 bash <(curl -s https://install.aeternity.io/install.sh)
 ```
 
+The latest argument is interpreted as version.
+The releases are published on [GitHub][releases].
+
 To install an older specific version use:
 ```bash
 bash <(curl -s https://install.aeternity.io/install.sh) 4.2.1
 ```
 
-See the documentation [here](https://github.com/aeternity/aeternity/blob/master/README.md) detailed instructions for manual installation.
+##### Additional options
+
+- `--no-prompt` - Disable confirmation prompt
+- `--delete` - Deleting the directory where aeternity node will be installed
+
+Example:
+
+```bash
+bash <(curl -s https://install.aeternity.io/install.sh) --no-prompt [version]
+```
+
+##### Additional parameters
+
+- `TARGET_DIR` - To override the path where aeternity node will be installed (default: `$HOME/aeternity/node`)
+
+Example:
 
+```bash
+TARGET_DIR=/some/dir bash <(curl -s https://install.aeternity.io/install.sh)
+```
+
+## Supported platforms
+
+* Ubuntu 16.04.3 LTS (x86-64)
+* Ubuntu 18.04 LTS (x86-64)
+* macOS High Sierra 10.13 (x86-64)
+* macOS Mojave 10.14 (x86-64)
+* macOS Catalina 10.15 (x86-64)
+
+## Dependencies that will be installed automatically
+
+* [Libsodium](https://download.libsodium.org/doc/)
+* [Openssl](https://www.openssl.org)
+* gmp
+
+See the documentation [here](https://github.com/aeternity/aeternity/blob/master/README.md) detailed instructions for manual installation.
```

Comment:
This sentence seem is not grammatically correct.

Reply:
For manual installation see the [detailed instructions](https://github.com/aeternity/aeternity/blob/master/README.md).

Reply:
a7b4ad4